src/ui/component/Learning.html

```diff
@@ -11,13 +11,17 @@
                 </a>
             {{/if}}
             {{#if course.state}}
-                {{#if course.type === 'elearning'}}
-                <a href="/courses/{{course.uid}}/do">Resume</a>
+                {{#if course.type === 'face-to-face'}}
```

Comment:
Are we not calling this `event` like yuksel proposed?

Reply:
The course is a face to face course which has events - had a follow on discussion around this with the business last week.

Reply:
Aha!